sources/LLREAD

```diff
@@ -1397,14 +1396,17 @@ Copyright (c) 1981-1988, 1990-1991, 1993, 2021 by Venue & Xerox Corporation.
                                                          (READ-EXTENDED-TOKEN STREAM])
 
 (CHARCODE.DECODE
-  [LAMBDA (C NOERROR)                                        (* ; 
-                                                           "Edited  1-Aug-2020 18:52 by rmk:")
-                                                             (* ; "Edited 18-Feb-87 22:03 by bvm:")
+  [LAMBDA (C NOERROR)                                   (* ; "Edited 21-Aug-2021 12:59 by rmk:")
```

Comment:
something odd with the pretty printer to get double font change  (C NOERROR)

Reply:
Dunno